### PR TITLE
Replace lib org.skyscreamer.jsonassert by org.sonar.test.JsonAssert.

### PR DESCRIPTION
**Pull request by simonbrandhof**
Replace lib org.skyscreamer.jsonassert by org.sonar.test.JsonAssert.

This class supports lenient comparison of datetime values.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ language: java
 
 jdk:
   - oraclejdk8
-  - openjdk7
```
Comment (teryk) [inline .travis.yml]: how do you use travis ? what's the link with replacing the jsonassert library ?

File: server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java [hunk 2]
```diff
@@ -63,8 +60,12 @@
 
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
-
-import java.util.*;
```
Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java]: why don't you have the java.util.* ?

File: server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java [hunk 3]
```diff
@@ -279,24 +280,6 @@ public DefaultIssue createManualIssue(String componentKey, RuleKey ruleKey, @Nul
     }
   }
 
-  // TODO result should be replaced by an aggregation object in IssueIndex
```
Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java]: what's the link with removing jsonassert ?

File: server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java (hunk 2 above)
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java]: You're right. I have to check my settings.

File: server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java (hunk 3 above)
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java]: In fact I use the travis build to track all test failures related to timezone coupling in tests. I should have done a second PR.

File: sonar-testing-harness/src/main/java/org/sonar/test/JsonAssert.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import com.google.gson.JsonParser;
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.ComparisonFailure;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * Assertion to compare JSON documents. Comparison is not strict:
```
Comment (teryk) [inline sonar-testing-harness/src/main/java/org/sonar/test/JsonAssert.java]: very nice documentation !

**Comment by teryk** [comment]
lgtm-big-lebowski

**Comment by teryk** [comment]
lgtm-pulp-fiction

**Pull request merged**